frontend/apps/web/app/(mgmt)/[account]/connections/page.tsx

```diff
@@ -59,7 +61,18 @@ function ConnectionTable(props: ConnectionTableProps): ReactElement {
 
   return (
     <div>
-      <DataTable columns={columns} data={connections} />
+      {connections.length == 0 ? (
```

Comment:
just curious why we need this. We have a button on all of these pages that a user can click to create a new entity. Why do we need it in two spots?

Could we instead change the table's empty state to just say click the new button or something?

Reply:
Just puts the button in front of the user in the middle of the screen and gives them guidance on what to do. We could remove the button and just keep the title and description.